packages/@dcl/inspector/src/lib/babylon/decentraland/sdkComponents/gltf-container.ts

```diff
@@ -1,13 +1,21 @@
 import * as BABYLON from '@babylonjs/core'
 import { GLTFFileLoader, GLTFLoaderAnimationStartMode } from '@babylonjs/loaders'
 import { GLTFLoader } from '@babylonjs/loaders/glTF/2.0'
-import { ComponentType, PBGltfContainer } from '@dcl/ecs'
+import { ComponentType, Entity, PBGltfContainer } from '@dcl/ecs'
 import { markAsCollider } from '../colliders-utils'
 import type { ComponentOperation } from '../component-operations'
 import { EcsEntity } from '../EcsEntity'
 import { SceneContext } from '../SceneContext'
 
-const sceneContextMap = new Map<string /*sceneId*/, WeakRef<SceneContext>>()
+type SceneId = string
+
+const sceneContextMap = new Map<
+  SceneId,
+  {
+    ctx: WeakRef<SceneContext>
+    entities: Set<Entity>
```

Comment:
Don't we have a map of entities already in the `SceneContext`? Why having another one here?